package.json

```diff
@@ -13,7 +13,7 @@
         "makemigrations": "docker-compose -f local.yml run django python manage.py makemigrations",
         "loaddata": "docker-compose -f local.yml run django python manage.py loaddata seed_data.json",
         "dumpdata": "docker-compose -f local.yml run django python manage.py dumpdata core.venue > core/fixtures/seed_data.json",
-        "bootstrap": "npm i && npm run build:docker && npm run build && npm run makemigrations && npm run loaddata"
+        "bootstrap": "npm i && npm run build:docker && npm run build && npm run loaddata"
```

Comment:
we run the command `makemigrations` when we change our models e.g. adding a field, deleting a model, etc..